### PR TITLE
Updated units.py

### PR DESCRIPTION
**Pull request by ghost**
Added some important physical constants ( Universal gas constant and Coulomb's constant )

### CODE REVIEW

**Comment by debugger22** [comment]
+1

It's always a good idea to commit changes on your local machine. Read our development workflow on wiki.

File: sympy/physics/units.py [hunk 1]
```diff
@@ -248,6 +248,8 @@ def free_symbols(self):
 G = gravitational_constant = Rational('6.67428') * ten**-11 * m**3 / kg / s**2
 u0 = magnetic_constant = 4*pi * ten**-7 * N/A**2
 e0 = electric_constant = 1/(u0 * c**2)
+Ke = coulombs_constant = 1/(4*pi * e0)
+R = gas_constant = Rational('8.3144621') * J / mol / K
```
Comment (bjodah) [inline sympy/physics/units.py]: Isn't the gas constant just defined as Avogadros number times Boltmannz's constant?

**Comment by gxyd** [comment]
Seems like `molar_gas_constant` has been added in `sympy/physics/units/definitions.py`. So we only need to add `coulombs_constant` in `sympy/physics/units/definitions.py`.

**Comment by gxyd** [comment]
This PR has been inactive for quite sometime, so we are going to close it and open an issue to reference it. We try to keep our pull request queue small and focused on active work.  We encourage you to reopen the pull request if and when you continue to work on this. Please contact us if you have any questions.

Thanks for contributing.

see https://github.com/sympy/sympy/wiki/Closing-pull-requests for our policies on closing pull request.

I've opened https://github.com/sympy/sympy/issues/13528 for this pull request so that we don't forget about it.

**Pull request closed without merging**